jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/PushBuilderImpl.java

```diff
@@ -23,6 +23,7 @@
 import org.eclipse.jetty.http.MetaData;
 import org.eclipse.jetty.util.URIUtil;
 
+@Deprecated(forRemoval = true)
```

Comment:
Orthogonal change?
Also, missing `since` attribute?

Reply:
yeah, this started off as a general 6.1 branch update.  We are mostly doing that without PRs, but the other changes needed review